models/models.go

```diff
@@ -25,3 +25,8 @@ type RelatedProducts struct {
 	Bom      string            `json:"bom"`
 	Products map[string]string `json:"products"`
 }
+
+type Script struct {
```

Comment:
The name can be more appropriate:
```suggestion
type ScriptInput struct {
```
Or
```suggestion
type ScriptParams struct {
```

Reply:
Done